virt/virt-web-console-overview.adoc

```diff
@@ -421,6 +421,9 @@ You can configure a VirtualMachine on the VirtualMachine details page.
 
 |xref:../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-snapshots_virt-web-console-overview[*Snapshots* tab]
 |Snapshot management
+
+|xref:../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-diagnostic_virt-web-console-overview[*Diagnostic* tab]
+|VirtualMachine diagnostics
```

Comment:
I recommend changing the description to "Status conditions and volume snapshot status" so that users know that this is different from, say, VM metrics or alerts.

Reply:
@apinnick I'm confused because I mention that exact phrase in my description. Do you mean to change the entire description content from:

```
You can view the status conditions and volume snapshot status on the *Diagnostic* tab.
```

to simply:

```
Status conditions and volume snapshot status.
```

If so, that doesn't seem to be in line with all of the other descriptions on the web console overview. I think I may need some clarity.

Reply:
Great, thanks for that clarification! Fixed.